common/djangoapps/student/models.py

```diff
@@ -179,6 +179,17 @@ class UserStanding(models.Model):
     standing_last_changed_at = models.DateTimeField(auto_now=True)
 
 
+class UserStandingConfig(ConfigurationModel):
+    disabled = models.TextField(
+        help_text="White-space separated list of usernames of users who should be disabled",
```

Comment:
I wouldn't worry about cache spamming. I think we're running with something like 4-6GB of cache on prod. Having a simple boolean entry for every user in the system barely dents that.